docs/release/v0.72.0-relnotes.md

```diff
@@ -0,0 +1,216 @@
+
+To create a user ID with a password, you have to visit the [Pbench dashboard](http://pbench.perf.lab.eng.bos.redhat.com/dashboard/) and click on `Login` in the upper right. That will pop up a `login/sign up` dialog through which you can create an account that will then allow you to generate a token (or login to the dashboard and look around). N.B. All data sent to the AWS Pbench Satellite or `pbench.app.intlab.redhat.com` “pass-through” server is owned by a `legacy` user: it’s all visible, but can’t be modified.
+
+The trouble is that this is a *very* temporary arrangement: we expect that very soon, you will be able to use Red Hat SSO for logging in and generating the document. The accounts created as above will go away, as will the `pbench-generate-token` script which is already deprecated. Any datasets submitted through this mechanism will therefore be orphaned, hence the imprecation to use this to kick the tires, not for storing "real" results that you don't want to lose. We are *NOT* planning to migrate any such results.
+
+
+New utilities to upload datasets as an authenticated user
+----
+The existing `pbench-move-results` command works in exactly the same way as before: it uses `ssh/scp` to copy the results to a pass-through server, but there is no notion of a user owning those
+results. Although we expect that to continue to be the main mode of operation for users of v0.72.0, we are moving towards a future where users will authenticate using SSO and that authenticated identity will become the owner of the datasets that are submitted by that user. The new commands `pbench-results-move` and `pbench-results-push` use an HTTP `PUT` to send results to a v1.0 Pbench Server which now provides a RESTful API to its services (`pbench-results-push` is used by the pass-through server to send the results to a Pbench server using a `legacy` user ID). The new commands will eventually supplant the existing `pbench-move-results`. Note that an early Pbench Server 1.0 integration with Pbench Agent introduced the `pbench-generate-token` command described above (available only Pbench Agent 0.71.0, 0.72.0) which is not compatible with the Pbench Server OIDC model that will allow us to use Red Hat SSO for authentication; the command is now deprecated and will be removed in the next Pbench Agent release.
+
+But we are not quite there yet: we may need to move to HTTPS in the Integration Lab and we need to finish setting up the authentication underpinnings in order to hook into the SSO infrastructure. Until that point, we encourage you to continue with pbench-move-results and the legacy interface to the server. If you have questions about any of this, ping us on GChat.
+
+
+
+Changelog (abbreviated - only the agent-side commits are included here)
+====
+For the full Changelog, see the link  near the top of these notes.
```

Comment:
Rather than referring the reader to the top of the notes, wouldn't it be just as easy to hyperlink it from here, as well?  (If you're concerned about replicating the link, I would refer the reader from the top to here....)